platform-dom/src/main/java/com/softicar/platform/dom/DomTestMarker.java

```diff
@@ -0,0 +1,10 @@
+package com.softicar.platform.dom;
+
+import com.softicar.platform.common.core.interfaces.IStaticObject;
+
+public enum DomTestMarker implements IStaticObject {
```

Comment:
We should probably chance all of those to singular.

Reply:
what exactly? `SECONDS_INPUT` and `MINUTES_INPUT`? well `SECOND_INPUT` looks like `2nd_INPUT`  so I would prefer plural, although your right, it then should be HOURS_INPUT